src/main/java/com/commercetools/sync/services/impl/ProductTypeServiceImpl.java

```diff
@@ -114,7 +119,7 @@ public ProductTypeServiceImpl(@Nonnull final BaseSyncOptions syncOptions) {
         return syncOptions.getCtpClient().execute(ProductTypeUpdateCommand.of(productType, updateActions));
     }
 
-    private CompletionStage<Optional<Map<String, AttributeMetaData>>> fetchAndCacheProductMetaData(
+    private CompletionStage<Optional<Map<String, AttributeMetaData>>> cacheAndFetchProductMetaData(
```

Comment:
One should fetch first to cache :)
I think the naming was better.

Reply:
You are right! 
https://github.com/commercetools/commercetools-sync-java/pull/316/commits/8e8f38b26775b89d2a000036440dc76e90848b2f